qiskit/pulse/schedule.py

```diff
@@ -386,6 +387,17 @@ def draw(self, dt: float = 1, style: Optional['SchedStyle'] = None,
             channels: A list of channel names to plot
             show_framechange_channels: Plot channels with only framechanges
 
+        Example:
+            ``scale`` parameter can be individually applied to channels when input is
+            dictionary format, otherwise single value is set to all channels.
+            If nothing specified, auto-scaling is applied.
```

Comment:
```suggestion
            If nothing is specified, auto-scaling is applied.
```

Reply:
Hi Taalexander,

Which range of auto-scaling is applied or unlimited, if nothing specified!?

Reply:
Good point, I've updated the suggestion.